linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_disable_nullok/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Disallow blank or null passwords in the PAM password-auth file'
+
```

Comment:
This rule is similar to rule `no_empty_passwords`. Wouldn't it be better to reuse the rule  `no_empty_passwords` instead of creating a new one? Please take a look into that at https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/rule.yml